compiler-no-html/package.js

```diff
@@ -7,7 +7,7 @@ Package.describe({
 
 Package.registerBuildPlugin({
   name: 'svelte-compiler',
-  use: ['svelte:core@1.2.1'],
+  use: ['svelte:core@1.23.4_1'],
```

Comment:
The latest version of `svelte:compiler-no-htlm` is `1.2.1`, so a version bump here is necessary if we want to publish a new version that will be deprecated.